docs/source/profiler.rst

```diff
@@ -600,6 +600,9 @@ Below is an breakdown of all the options.
     * **multiprocess** - Option to enable multiprocessing. Automatically selects the optimal number of processes to utilize based on system constraints.
 
       * is_enabled - (Boolean) Enables or disables multiprocessing
+      
+    * **sample_ratio** - The ratio from 0 to 1 of how much of the input data to sample. Default value set to 0.2.
```

Comment:
```suggestion
    * **sample_ratio** - A percentage ranging from 0 to 1 indicating how much input data to sample. Default value set to 0.2.
```

Reply:
Could you submit the same commit suggestion again? This commit suggestion is outdated after I resolved other commit suggestions.

Reply:
done